src/validateInvocation.test.ts

```diff
@@ -30,12 +30,16 @@ it('auth error', async () => {
 
   const executionContext = createMockExecutionContext({
     instanceConfig: {
-      clientId: 'INVALID',
-      clientSecret: 'INVALID',
+      cybereasonId: 'INVALID',
+      cybereasonPassword: 'INVALID',
+      cybereasonHost: 'INVALID',
+      cybereasonPort: 'INVALID',
     },
   });
 
+  // TEMP: Change assertion pass test and push for draft PR
+
   await expect(validateInvocation(executionContext)).rejects.toThrow(
-    IntegrationProviderAuthenticationError,
+    IntegrationProviderAPIError,
```

Comment:
If the host and port is invalid, it will throw an API error instead because you can't verify authentication without a valid host and port. Is this acceptable or is it required to assert IntegrationProviderAuthenticationError here, and if so, is it okay to push the server's credentials?

Reply:
Makes sense! Just add that to the comment somewhere so that we note down the reason behind this change.